ui/app/src/modules/Preview/Tools/ContentTree.tsx

```diff
@@ -471,13 +486,15 @@ export default function ContentTree() {
       >
         {data.selected && (
```

Comment:
This condition is no longer required with Suspencified, is it?
remove the conditional on the handlePrevious prop and handle the logic of whether there are previous on the handler and the child component - since it has access to the resource, which contains “previous”, it can do the necessary checks.